misc/scripts/install-local.sh

```diff
@@ -104,8 +104,10 @@ if [[ -n "$pacdeps" ]]; then
     for i in "${pacdeps[@]}"
     do
         fancy_message info "Installing $i"
+        echo "PACSTALL_DEPENDS=true" | sudo tee /tmp/pacstall-pacdeps
         sudo pacstall -P -I "$i"
         pacdepslist+=($i)
+        sudo rm -f /tmp/pacstall-pacdeps
```

Comment:
You are writing to the file, and removing the in the same code?

Reply:
After installation of a package so that the metadata logger can log it as a dependency, then delete it

Reply:
IF the logger logs it, and then deletes it, whats the use? Ultimately there is no log of it left.

Reply:
? It's to check if a package is installed as a dependency. For instance, running `sudo pacstall -I soundux` will install `pipewire` as a dependency. So it makes the file, the logger detects it and shows that is was installed as a dependency when running `pacstall -Qi pipewire`. Then it deletes the file

Reply:
This part of the code is logging it (writing it to the file), the detecting part starts from L233, so if the logger itself delets the file, then how will the detector (which comes later) detect it?

Reply:
Because running `sudo pacstall -I pkg` will at the end, look for the file that signifies that the package was installed as a dependency. Then it will delete after `sudo pacstall -I pkg` is done. It's running two pacstall instances

Reply:
Omg, well it got way more complex.